src/editors/containers/VideoUploadEditor/index.jsx

```diff
@@ -11,77 +12,73 @@ import * as hooks from './hooks';
 import messages from './messages';
 import * as editorHooks from '../EditorContainer/hooks';
 
-export const VideoUploader = ({ onUpload, errorMessage }) => {
-  const { textInputValue, setTextInputValue } = hooks.uploader();
-  const onURLUpload = hooks.onVideoUpload();
+const URLUploader = ({ intl }) => (
```

Comment:
You can use `const intl = useIntl()` here. I think the advice against that is outdated since this uses a newer version of platform now which includes that.